content/download.md

```diff
@@ -26,64 +26,67 @@ limitations under the License.
 
-{% comment %}
-
-TODO: uncoment this when we have official releases of Rya
-
 {{ site.data.project.name }} is released as a source artifact, and also through Maven.
 
 ### Source releases
 
-Release          | Date       | Commit   | Download
-:--------------- | :--------- | :------- | :-------
+Release          | Date       | Commit   | Download | Notes
+:--------------- | :--------- | :------- | :------- | :----
 {% for post in site.categories.release %}{% comment %}
 {% endcomment %}{% if post.fullVersion %}{% comment %}
 {% endcomment %}{% assign v = post.fullVersion %}{% comment %}
 {% endcomment %}{% else %}{% comment %}
 {% endcomment %}{% capture v %}apache-{{ site.data.project.unix_name }}-{{ post.version }}{% endcapture %}{% comment %}
 {% endcomment %}{% endif %}{% comment %}
 {% endcomment %}{% if forloop.index0 < 2 %}{% comment %}
-{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua?filename={{ site.data.project.unix_name }}/{{ v }}{% endcapture %}{% comment %}
+{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua?filename={{ site.data.project.incubator_slash_name }}/{{ v }}{% endcapture %}{% comment %}
 {% endcomment %}{% assign q = "&action=download" %}{% comment %}
-{% endcomment %}{% assign d = "https://www.apache.org/dist" %}{% comment %}
+{% endcomment %}{% assign d = "https://www.apache.org/dist/incubator" %}{% comment %}
 {% endcomment %}{% else %}{% comment %}
 {% endcomment %}{% capture p %}http://archive.apache.org/dist/incubator/{{ site.data.project.unix_name }}/{{ v }}{% endcapture %}{% comment %}
 {% endcomment %}{% assign q = "" %}{% comment %}
 {% endcomment %}{% assign d = "https://archive.apache.org/dist/incubator" %}{% comment %}
 {% endcomment %}{% endif %}{% comment %}
-{% endcomment %}<a href="{{ site.baseurl }}/history.html#{{ post.tag }}">{{ post.version }}</a>{% comment %}
+{% endcomment %} <a href="{{ post.url }}">{{ post.version }}</a>{% comment %}
 {% endcomment %} | {{ post.date | date_to_string }}{% comment %}
-{% endcomment %} | <a href="https://github.com/apache/{{ site.data.project.unix_name }}/commit/{{ post.sha }}">{{ post.sha }}</a>{% comment %}
-{% endcomment %} | <a href="{{ p }}/{{ v }}-src.tar.gz{{ q }}">tar</a>{% comment %}
+{% endcomment %} | <a href="{{ site.data.project.source_repository_mirror }}/commit/{{ post.sha }}">{{ post.sha }}</a>{% comment %}
+// Currently the mirrors have no signitures, so use dist.apache.org:
+{% endcomment %} | {% if post.filename %} <a href="{{ p }}/{{ post.filename }}{{ q }}">download</a>{% comment %}  
+{% endcomment %} , [signatures](https://dist.apache.org/repos/dist/release/{{ site.data.project.incubator_slash_name }}/{{ v }}){% comment %}
```

Comment:
Replace "signature" link with 3 separate direct links to .asc .md5 and .sha1 files